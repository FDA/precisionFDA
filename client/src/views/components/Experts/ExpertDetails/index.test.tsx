import React from 'react'
import { shallow } from 'enzyme'
import { Provider } from 'react-redux'
import configureMockStore from 'redux-mock-store'

import { ExpertDetails } from '.'

const mockStore = configureMockStore()
const store = mockStore({})

describe('ExpertDetails test', () => {
  it('should render ExpertDetails', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <ExpertDetails expert={() => {}} />
      </Provider>,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
