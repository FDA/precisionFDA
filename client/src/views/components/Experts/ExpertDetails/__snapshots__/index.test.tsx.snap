// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpertDetails test should render ExpertDetails 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Object {
        "addNestedSub": [Function],
        "getListeners": [Function],
        "handleChangeWrapper": [Function],
        "isSubscribed": [Function],
        "notifyNestedSubs": [Function],
        "trySubscribe": [Function],
        "tryUnsubscribe": [Function],
      },
    }
  }
>
  <ExpertDetailsComponent
    expert={[Function]}
  />
</ContextProvider>
`;
